Bottom: 8570e52423c23dfec5a51706108b7bf2e0545543
Top:    2c5ef05bd4b4b7144d2adeebce08ba9fa5b6b3dd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 02:18:22 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
